server/server_test.go

```diff
@@ -41,16 +39,21 @@ func (s *ServerSuite) SetUpSuite(c *C) {
 
 	// Set up the database
 	var err error
-	s.Session, err = mgo.Dial("localhost")
-	util.CheckErr(err)
-	s.Database = s.Session.DB(config.DatabaseName)
+	s.Connection = new(MongoConnection)
+	err = s.Connection.Connect("localhost")
+
+	if err != nil {
+		panic(err)
+	}
```

Comment:
This is what `util.CheckErr(err)` is for (see deleted code in the diff).  It's cleaner.